monai/bundle/scripts.py

```diff
@@ -546,10 +546,9 @@ def _get_all_bundles_info(
     repo: str = "Project-MONAI/model-zoo", tag: str = "hosting_storage_v1", auth_token: str | None = None
 ) -> dict[str, dict[str, dict[str, Any]]]:
     if has_requests:
-        if tag == "hosting_storage_v1":
+        request_url = f"https://raw.githubusercontent.com/{repo}/{tag}/models/model_info.json"
+        if requests_get(request_url).status_code != 200:
```

Comment:
this means it'll double the number of requests for every default model info read?